config/config.go

```diff
@@ -0,0 +1,33 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/tkanos/gonfig"
+)
+
+type Configuration struct {
+	DbHostname string `json:"DB_HOSTNAME"`
+	DbUser     string `json:"DB_USER"`
+	DbPassword string `json:"DB_PASSWORD"`
+	DbName     string `json:"DB_NAME"`
+}
+
+func GetConfig(params ...string) Configuration {
+
+	configuration := Configuration{}
+
+	env := "dev"
+
+	if len(params) > 0 {
+
+		env = params[0]
+
+	}
```

Comment:
Empty lines are not encouraged by Go.

Are needed when you need to separate certain blocks of code for better readability, but in this case, are not necessary.

```suggestion
	if len(params) > 0 {
		env = params[0]
	}
```